modules/govuk/manifests/app/config.pp

```diff
@@ -390,7 +390,29 @@
     maxsize => '100M',
   }
 
-  if $health_check_path != 'NOTSET' {
+  if $has_readiness_health_check {
+    include icinga::client::check_json_healthcheck
+
+    $healthcheck_desc      = "${title} app healthcheck not ok"
+
+    if $health_check_custom_doc {
+      $healthcheck_doc_slug = regsubst($healthcheck_desc, ' ', '-', 'G')
+    } else {
+      $healthcheck_doc_slug = 'app-healthcheck-not-ok'
+    }
```

Comment:
Do we need this conditional?
Can't we simply have `$healthcheck_doc_slug = regsubst($healthcheck_desc, ' ', '-', 'G')`?

Reply:
Sometimes the condition is true, sometimes it's false.  I think usually it's false, but there are a few apps with custom docs.

Reply:
Yes, what I meant is if we only have the regexp replacement function, when there aren't custom docs, the slug would become "app-healthcheck-not-ok" because of the description. But I see now that it would become "#{title}-app-healthcheck-not-ok" instead, which I guess is not ok!